exporters/trace/fluentforward/fluentforward.go

```diff
@@ -0,0 +1,199 @@
+package fluentforward
+
+import (
+	"context"
+	"errors"
+	"log"
+	"net"
+	"time"
+
+	apitrace "go.opentelemetry.io/otel/api/trace"
+	export "go.opentelemetry.io/otel/sdk/export/trace"
+	sdktrace "go.opentelemetry.io/otel/sdk/trace"
+	"go.opentelemetry.io/otel/api/global"
+	"go.opentelemetry.io/otel/label"
+	"github.com/vmihailenco/msgpack/v5"
+)
+
+type ffSpan struct{
+	_msgpack struct{}            `msgpack:",asArray"`
+	Tag string                   `msgpack:"tag"`
+	Ts int64                     `msgpack:"ts"`
+	SpanData spanData						 `msgpack:"spanData"`
+}
+
+type spanData struct{
+	SpanContext spanContext         `msgpack:"spanContext"`
+	ParentSpanId string             `msgpack:"parentSpanId"`
+	SpanKind apitrace.SpanKind      `msgpack:"spanKind"`
+	Name string                     `msgpack:"name"`
+	StartTime int64                 `msgpack:"startTime"`
+	StatusMessage string            `msgpack:"statusMessage"`
+	EndTime int64                   `msgpack:"endTime"`
+	Attrs map[label.Key]interface{} `msgpack:"attrs"`
+	DroppedAttributeCount int       `msgpack:"droppedAttributesCount"`
+	DroppedMessageEventCount int    `msgpack:"droppedMessageEventCount"`
+	DroppedLinkCount int            `msgpack:"droppedLinkCount"`
+	StatusCode string               `msgpack:"statusCode"`
+	Links []link                    `msgpack:"links"`
+}
+
+type spanContext struct{
```

Comment:
why the spancontext is needed? shouldn't it be just like in a proto and be part of a Span?

Reply:
That's the way it has been implemented in go, I kept it the same based on [this](https://pkg.go.dev/go.opentelemetry.io/otel@v0.13.0/api/trace#SpanContext)

Reply:
Go API is optimized for coding. I suggest to align with the proto that is intended for exporting and consumption.